src/ragger/conftest/base_conftest.py

```diff
@@ -0,0 +1,205 @@
+import pytest
+from typing import Optional
+from pathlib import Path
+from ragger.firmware import Firmware
+from ragger.backend import SpeculosBackend, LedgerCommBackend, LedgerWalletBackend
+from ragger.navigator import NanoNavigator, StaxNavigator
+from ragger.utils import find_project_root_dir
+from ragger.logger import get_default_logger
+
+from ragger.conftest import configuration as conf
```

Comment:
To avoid passing through `ragger/conftest/__init__.py` which could end up into circular import, I think relative import is better when inside the same module.
```suggestion
from . import configuration as conf
```

Reply:
Done